pages/servers/[serverId]/polls/results/[pollId].tsx

```diff
@@ -29,6 +29,10 @@ const PollResults: NextPage = () => {
 
   const isLoadingPoll = !pollData && !error
 
+  function returnToServer() {
+    router.push(`/servers/${currentServer?.id ?? ''}`)
```

Comment:
This works great; found a similar bug that maybe needs this fix too.

Login, click the Garage Server, then click The Garage link in the breadcrumbs, it should blink and logout.

Considering we're already on that page, maybe it makes sense just to remove the link (or even turn it into a Back button)